swf/src/write.rs

```diff
@@ -2225,7 +2226,7 @@ impl<W: Write> Writer<W> {
         Ok(())
     }
 
-    fn write_define_text(&mut self, text: &Text) -> Result<()> {
+    fn write_define_text(&mut self, text: &Text, version: u8) -> Result<()> {
```

Comment:
Nice to avoid the "boolean trap" by asking for the version as a number.
I guess this is a really internal function, so a note saying that only versions 1 and 2 are valid is not really necessary.

Reply:
We do this in other similar write methods too, so I just copied the pattern :D

Reply:
Sure, it's perfectly fine this way.